hooks/useNodeJsContributorsApi.tsx

```diff
@@ -0,0 +1,181 @@
+import useSWR from 'swr';
+import { NodeJSContributorsApiKey } from '../constants/swr';
+
+const LIMIT_CONTRIBUTORS = 5;
+const CONTRIBUTORS_API_URI = `https://api.github.com/repos/nodejs/node/contributors?per_page=${LIMIT_CONTRIBUTORS}`;
```

Comment:
I feel we could have a dedicated constants file for all these sorts of constants, including API URL constants as we had on nodejs.dev

Reply:
(To have a single point for all constants, which makes it easier to find)